sciencebeam/transformers/grobid_service.py

```diff
@@ -23,32 +24,40 @@ def get_logger():
 def start_service_if_not_running():
   service_wrapper.start_service_if_not_running()
 
-def grobid_service(base_url, path, start_service=True, field_name=None):
+def run_grobid_service(x, base_url, path, start_service=True, field_name=None):
```

Comment:
`x` seems too vague here, especially when the others args are quite descriptive.

Reply:
Okay, you are right. It's the item being processed (which can either be a tuple of (filename, pdf content) or just the pdf content). The idea was initially that `grobid_service` creates a function that can be directly used with `beam.Map`. In that case we can only pass one value (which could be a tuple). I renamed it to the not much more descriptive name `item`. But I added some documentation. The function and use could probably be refactored. As part of this development I just wanted to allow the `path` to be specified per call.